packages/resolver/test/truffle.js

```diff
@@ -0,0 +1,43 @@
+const assert = require("assert");
+const path = require("path");
+
+// Use transpiled output to properly test the path manipulation
+// truffle resolver utilizes
+const { Truffle } = require("../dist/lib/sources/truffle");
+const resolver = new Truffle({});
+
+describe("truffle resolve [ @win ]", function () {
```

Comment:
This `"@win"` tag doesn't do anything at the moment, right?  It's for later?

Reply:
Oh, good catch. This PR grabbed commits from the `windows/runner` branch and I missed that. The tag allows for filtering tests for specific runs in that branch.